assets/dev/js/editor/regions/navigator/element.js

```diff
@@ -109,6 +109,10 @@ export default class extends Marionette.CompositeView {
 	}
 
 	hasChildren() {
+		if ( elementor.hooks.applyFilters( 'elementor/editor/navigator/element/has-children', false, this.model ) ) {
```

Comment:
There is a issue, it may work not as assumed, the following code does not works:
```javascript
	hasChildren() {
		const containerId = this.model.get( 'id' );

		if ( containerId ) {
			const container = elementor.getContainer( containerId );

			if ( container ) {
				return !! container.children.length;
			}
		}

		return 'widget' !== this.model.get( 'elType' );
	}
```

it works only when editor loads, when you create a nested tab, there is a tick between render, and creation of new containers.

how do you wish face the issue?